cmd/leo/destroyrunner/destroyrunner.go

```diff
@@ -0,0 +1,61 @@
+package destroyrunner
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spf13/cobra"
+
+	"terraform-provider-iterative/iterative/aws"
+	"terraform-provider-iterative/iterative/azure"
+	"terraform-provider-iterative/iterative/gcp"
+	"terraform-provider-iterative/iterative/kubernetes"
+
+	"terraform-provider-iterative/task/common"
+)
+
+type Options struct {
+	Name string
+}
+
+func New(cloud *common.Cloud) *cobra.Command {
+	o := Options{}
+
+	cmd := &cobra.Command{
+		Use:    "destroy-runner <identifier>",
+		Short:  "Destroy a CML runner",
+		Long:   ``,
+		Hidden: true,
+		Args:   cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return o.Run(cmd, args, cloud)
+		},
+	}
+
+	cmd.Flags().StringVar(&o.Name, "name", "", "needed for Google Cloud runners") // FIXME: it shouldn't
+
+	return cmd
+}
+
+func (o *Options) Run(cmd *cobra.Command, args []string, cloud *common.Cloud) error {
+	d := &schema.ResourceData{}
+
+	d.Set("region", cloud.Region)
+	d.Set("name", o.Name)
```

Comment:
Thanks, linter! Let's check for errors that will never happen:

```suggestion
	for key, value := map[string]string{
		"region": cloud.Region,
		"name": o.Name
	} {
		if err := d.Set(key, value); err != nil {
			return err
		}
	}
```